components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -22,5 +24,18 @@
 </template>
 
 <script lang="ts" setup>
+import { ref, type Ref } from 'vue'
 import SVGLookingGlass from '~/assets/icons/looking-glass.svg'
+import type { HealthcareProfessional } from '~/typedefs/gqlTypes'
+import { getHealthcareProfessionalById } from '~/stores/healthcareProfessionalsStore'
+
+const requestedHealthcareProfessionalId: Ref<string> = ref('')
+const healthcareProfessionalById: Ref<HealthcareProfessional[] | []> = ref([])
```

Comment:
is this being used by anything? I think we can remove it

Reply:
I set the value like this:
```
healthcareProfessionalById.value = await getHealthcareProfessionalById(requestedHealthcareProfessionalId.value)
```

on line 39

Reply:
I see it being set, but I don't see it used anywhere after it's set

Reply:
It's set already for the PR after this, when we implement the UI, just wanted to make it a bit easier for the person/myself implementing the when working on the UI ticket. 😅

Reply:
Ahhh ok, that's worth putting a comment on in the future since it's unused code. 👍